winui/ComboBox/Highlighting.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: post
+title: Highlighting Feature in WinUI Combobox control | Syncfusion
+description: Learn here all about Highlighting Feature in Syncfusion WinUI Combobox control into WinUI application.
+platform: winui
+control: SfComboBox
+documentation: UG
+---
+
+# Highlighting in WinUI ComboBox(SfComboBox)
+
+Highlight matching characters in a suggestion list to pick an item with more clarity. There are two ways to highlight the matching text:
+
+* **Highlight beginning text**
+* **Highlight all occurance of search text**
+
+
+The text highlight can be indicated with various customizing styles by enabling the below properties. They are
```

Comment:
The text highlight can be indicated with various customizing styles by enabling the following properties.